### DISCUSSION TITLE
how  to call mysql user defined store procedure

### DISCUSSION THREAD

**Discussion by njalan** [trinodb/trino#23170] [Q&A]
I am using trino 397 and does it support all mysql user defined store procedure?
I tried to use call but got below error:
Procedure not registered

**Comment by hashhar** [comment]
Only SQL Server connector has https://trino.io/docs/current/connector/sqlserver.html#procedure-varchar-table which allows to execute procedures defined in SQL Server.

For MySQL you can try to use https://trino.io/docs/current/connector/mysql.html#system-execute-query or https://trino.io/docs/current/connector/mysql.html#query-varchar-table to execute a remote procedure but it may not work.

But neither of these might be available in Trino 397 which is now ~2 years old.
